packages/hub/services/contract-subscription-event-handler.ts

```diff
@@ -21,6 +21,12 @@ export const CONTRACT_EVENTS = [
     eventName: 'MerchantClaim',
     taskName: 'notify-merchant-claim',
   },
+  {
+    abiName: 'prepaid-card-market-v2',
```

Comment:
abi is `prepaid-card-market-v-2` I think, but `v2` is used elsewhere.

Reply:
ugh fixed thanks